src/pages/_lend/LendPage.tsx

```diff
@@ -2,25 +2,30 @@ import StellarIcon from '../../images/Stellar_Symbol.png';
 import USDCIcon from '../../images/usdc.svg';
 import { LendableAssetCard } from './LendableAssetCard';
 
+import XLMPoolContract from '@contracts/loan_pool';
+import USDCPoolContract from '@contracts/usdc_pool';
+
 const currencies = [
-  {
-    name: 'USD Coin',
-    symbol: 'USDC',
-    icon: USDCIcon.src,
-  },
   {
     name: 'Stellar Lumen',
     symbol: 'XLM',
     icon: StellarIcon.src,
+    contractClient: XLMPoolContract,
+  },
+  {
+    name: 'USD Coin',
+    symbol: 'USDC',
+    icon: USDCIcon.src,
+    contractClient: USDCPoolContract,
   },
 ];
 
```

Comment:
Eikö nää pitänyt nyt tulla sieltä currencies.ts filusta?

Reply:
Oho hyvä huomio, kyllä pitäis 👍